.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+- [ ] **Submitter**: Include the JIRA issue number in the PR description
```

Comment:
you can do this with template PR and Issue files (https://github.com/blog/2111-issue-and-pull-request-templates)